sway/criteria.c

```diff
@@ -189,7 +189,7 @@ static bool criteria_matches_view(struct criteria *criteria,
 	if (criteria->title) {
 		const char *title = view_get_title(view);
 		if (!title) {
-			return false;
+			title = "";
```

Comment:
I don't think this is correct, this changes the title of windows when trying to match it to a criteria.

This is not the right place to do this.

Reply:
This doesn't change the title of windows. It just falls back to a value which can be matched.